packages/editor/src/Graf/Graf.tsx

```diff
@@ -107,18 +107,29 @@ export const Graf: React.FC<{ node: ParserField }> = ({ node }) => {
           <ActiveNode
             readonly={readonly}
             onDuplicate={(nodeToDuplicate) => {
-              const { ...rest } = node;
+              const { ...rest } = nodeToDuplicate;
               const newName = nodeToDuplicate?.name + "Copy";
+              const newId = generateNodeId(
```

Comment:
a może lepiej to rozwiązać w ten sposób, że jeśli jest już node skopiowany ale ktoś chce skopiować dwa razy, to zrobić "Copy" "Copy1" "Copy2"

Reply:
a co z umieszczeniem tego node'a? Chyba się nakładają teraz.
Mogę zmienić po prostu nie było to w tikecie opisane